src/muiComponents/Swap/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Paper, TextField, Typography } from "@mui/material"
+import React, { ReactElement } from "react"
+
+interface Props {
+  inputValue?: number
+}
+
+export default function SwapPage({ inputValue }: Props): ReactElement {
+  return (
+    <Box>
+      <Paper>
+        Swap
+        <Box display="flex">
+          <Typography>From</Typography>
+          <Typography>Balance:{inputValue}</Typography>
```

Comment:
Should it be `Balance: {inputValue}`?

Reply:
Because we will work this part on another branch so, I removed this component and resolve this conversation.